src/Common.Dotnet.CsWinRT.props

```diff
@@ -33,4 +34,11 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Windows.CsWinRT" />
   </ItemGroup>
+
+  <!-- this may need to be removed on future CsWinRT upgrades-->
```

Comment:
where do we document these things?

Reply:
i.e. leaving it here _ensures_ it will be forgotten forever

Reply:
Good callout, addressed inside Directory.packages.props now above CsWinRT version so on upgrade, it will be seen